namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-to-probation-update-prod/resources/dynamodb.tf

```diff
@@ -8,7 +8,7 @@ module "message_dynamodb" {
   infrastructure-support       = var.infrastructure-support
   is-production                = "true"
   namespace                    = var.namespace
-  autoscale_max_read_capacity  = 500
+  autoscale_max_read_capacity  = 5000
   autoscale_max_write_capacity = 20
 
```

Comment:
with only 11,000 rows with a 10 second cron that reads some rows we reach that initial limit straight away, so going for a much bigger number to see where it settles at